### ISSUE TITLE
Can't turn the plugin off

### ISSUE THREAD

**Issue by shadshack** [CinderBlocc/OnlineStatusChanger#1]
When unchecking the "Enable status override" button it doesn't disable the plugin from overwriting the status, and keeps whatever the last status that was set to. There's no way I can find to revert to letting the game handle rich presence on it's own. Even uninstalling the plugin and deleting it's settings file doesn't fix it.

**Comment by CinderBlocc** [comment]
All the plugin does is create a GUI interface for the online status override cvars that are built into BakkesMod. Unfortunately those cvars are really finnicky and don't often do what you intend. My only suggestion for a solution is to keep toggling the settings and just hope that it sticks one of those times. Sorry!

**Comment by shadshack** [comment]
Thanks for the tip. I didn't realize it was using BM cvars. I ended up getting it to work by uninstalling the plugin and opening the BakkesMod console with F6, and running the command ' cl_online_status_override "" '. (I think that accomplishes the same thing that you suggested, just without the GUI.) I did try that before I posted this issue, but was just sitting at the main menu and didn't realize the rich text wouldn't actually update until something changed in-game. Had to load into Training for it to update.
